File: src/main/java/br/com/jiratorio/domain/request/HolidayRequest.java
package br.com.jiratorio.domain.request;

import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Pattern;
import lombok.Data;

@Data
public class HolidayRequest {

    @NotBlank
    @Pattern(regexp = "[0-9]{0,2}/[0-9]{0,2}/[0-9]{0,4}")
    private String date;

    @NotBlank
    private String description;

}

File: src/main/java/br/com/jiratorio/domain/request/HolidayRequest.kt
package br.com.jiratorio.domain.request

import com.fasterxml.jackson.annotation.JsonFormat
import java.time.LocalDate
import javax.validation.constraints.NotBlank

data class HolidayRequest(

    @JsonFormat(pattern = "dd/MM/yyyy")
    var date: LocalDate,

    @field:NotBlank
    var description: String

)
